spec/system/user_spec.rb

```diff
@@ -1,125 +1,127 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 RSpec.describe 'ユーザ登録・ログイン・ログアウト機能', type: :system do
-    before do
-        @user = FactoryBot.create(:user)
-        @admin_user = FactoryBot.create(:admin_user)
-    end
+  before do
+    @user = FactoryBot.create(:user)
+    @admin_user = FactoryBot.create(:admin_user)
```

Comment:
RSpecだとインスタンス変数を使うよりletを使うことが多いです！
```suggestion
    let(:admin_user) { create(:admin_user) }
```
と記載して admin_user というローカル変数を記載すれば 対応する`{ }`内の処理が実行されて返り値が変数に入ります。
`let!(:admin_user) { create(:admin_user) }`と!をつけると変数を宣言しなくてもコード実行時に即時実行されるようになります。（beforeで定義するような感覚です。）

Reply:
理解に時間がかかりそうなので別途時間をとって確認しようと思います。